region/norfair/east.json

```diff
@@ -1742,7 +1742,7 @@
                     "heatProof",
                     "canHeatRun"
                   ]},
-                  "reset",
+                  {"resetRoomAtNode": [1]},
```

Comment:
Needs reset at opposite side.
Also, traversing from 1 to 2 requires a reset at 1.